variety/Util.py

```diff
@@ -890,12 +897,67 @@ def copy_with_replace(from_path, to_path, search_replace_map):
         with open(to_path + ".partial", "w") as file:
             file.write(data)
             file.flush()
-        os.rename(to_path + ".partial", to_path)
+        shutil.move(to_path + ".partial", to_path)
 
     @staticmethod
     def get_exec_path():
         return os.path.abspath(sys.argv[0])
 
+    @staticmethod
+    def set_windows_registry_key(path, name, value, dtype=None, master='HKEY_CURRENT_USER'):
+        import winreg
+
+        # Type is only used if creating a new value
+        if not dtype or isinstance(value, str):
+            dtype = winreg.REG_SZ
+        elif isinstance(value, int):
+            dtype = winreg.REG_DWORD
+        else:
+            dtype = winreg.REG_SZ
+
+        # Translate key name to winreg key name
+        registry = winreg.HKEY_CURRENT_USER
+        if master == 'HKEY_CURRENT_USER':
+            registry = winreg.HKEY_CURRENT_USER
+        elif master == 'HKEY_LOCAL_MACHINE':
+            registry = winreg.HKEY_LOCAL_MACHINE
+        elif master == 'HKEY_USERS':
+            registry = winreg.HKEY_USERS
```

Comment:
If no hives other than HKEY_CURRENT_USER are used we can probably remove those cases. AFAICT [HKEY_LOCAL_MACHINE](https://www.stigviewer.com/stig/windows_10/2015-11-30/finding/V-63593) shouldn't be writable from an unprivileged account anyways.